src/bundle/Resources/public/js/scripts/helpers/object.instances.js

```diff
@@ -0,0 +1,17 @@
+(function(global, doc, ibexa) {
+    const set = (domElement, instance) => {
```

Comment:
I'm not sure, but I would probably prefer more verbose names e.g. `setInstance` etc. given these are functions and not object methods. WDYT?🤔